hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-globalpolicygenerator/src/main/java/org/apache/hadoop/yarn/server/globalpolicygenerator/GlobalPolicyGenerator.java

```diff
@@ -155,6 +163,16 @@ protected void serviceStart() throws Exception {
           DurationFormatUtils.formatDurationISO(scCleanerIntervalMs));
     }
 
+    // Schedule ApplicationCleaner service
+    long appCleanerIntervalMs = config.getLong(YarnConfiguration.GPG_APPCLEANER_INTERVAL_MS,
```

Comment:
I know is a backport but... Can we use getTimeDuration?

Reply:
I will improve this part of the code.